src/classes/Main.cls

```diff
@@ -0,0 +1,12 @@
+public with sharing class Main {
+	
+	 public static void main() {
+	 	
+	 	String x = '';
+	 	if (x=='5003000000D8cuI') {
```

Comment:
Record IDs change between environments, so this logic will fail when deployed to a different sandbox or org. Please retrieve the required ID dynamically and avoid hardcoding.